.ci/check-release-sigs

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
I think you need `set -e` otherwise it won't actually fail the build.

Reply:
Good catch!